misc/scripts/checks.sh

```diff
@@ -553,8 +553,29 @@ function lint_priority() {
     return "${ret}"
 }
 
+function lint_license() {
```

Comment:
Add a way to specify custom licenses, which I believe PKGBUILDs do it like `custom:foobar`.